modules/grafana/manifests/datasources.pp

```diff
@@ -29,7 +29,7 @@
     \"name\":\"CloudWatch\",
     \"type\":\"cloudwatch\",
     \"access\":\"proxy\",
-    \"jsonData\":{\"authType\":\"arn\", \"defaultRegion\":\"eu-west-1\", \"assumeRoleArn\":\"\", \"timeField\":\"@timestamp\"}
+    \"jsonData\":{\"authType\":\"arn\", \"defaultRegion\":\"eu-west-1\", \"assumeRoleArn\":\"\", \"timeField\":\"@timestamp\", \"customMetricsNamespaces\": \"AWS/AmazonMQ\"}
```

Comment:
Probs just a q for my understanding: Is this custom namespace doing anything, given we add it in on the templating anyway?

Reply:
According to the docs this is what makes the metrics available in the online editor UI. 
There's some weird caching going on that means it's difficult to test this with/without the line - I can't get rid of it in the 'Edit datasource' UI now, even after deploying a revision without it.

So I suggest we err on the side of caution and keep it in?

Reply:
I've now managed to test this without the custom namespace in 'Datasources' - and it still seems to work, so I've removed it from the code.